qa-templates-checker.sh

```diff
@@ -0,0 +1,64 @@
+#! /usr/bin/env bash
```

Comment:
Please remove this from the repo, place in a [gist](https://gist.github.com/), and add the link to the QA steps.

Reply:
Done